src/uproot/models/TDatime.py

```diff
@@ -0,0 +1,78 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+"""
+This module defines versioned models for ``TDatime``.
+"""
+
+from __future__ import absolute_import
+
+import struct
+
+import numpy
+
+import uproot
+
+_tdatime_format1 = struct.Struct(">I")
+
+
+class Model_TDatime(uproot.model.Model):
+    """
+    A versionless :doc:`uproot.model.Model` for ``TDatime``.
+    """
+
+    def read_numbytes_version(self, chunk, cursor, context):
+        pass
+
+    def read_members(self, chunk, cursor, context, file):
+        (
+            self._members["fDatime"],
+        ) = cursor.fields(chunk, _tdatime_format1, context)
+
+    @classmethod
+    def strided_interpretation(
+        cls, file, header=False, tobject_header=True, breadcrumbs=(), original=None
+    ):
+        members = []
+        if header:
+            members.append(("@num_bytes", numpy.dtype(">u4")))
+            members.append(("@instance_version", numpy.dtype(">u2")))
+        members.append(("fDatime", numpy.dtype(">u4")))
+        return uproot.interpretation.objects.AsStridedObjects(
+            cls, members, original=original
+        )
+
+    @classmethod
+    def awkward_form(
+        cls, file, index_format="i64", header=False, tobject_header=True, breadcrumbs=()
+    ):
```

Comment:
I wasn't able to figure out how to provide test coverage for this method.

Reply:
It's okay to not provide coverage for this. It's going to be replaced (or perhaps just modified) in the AwkwardForth project, which is coming up soon. We can address these methods at that time.